sunpy/net/dataretriever/sources/eve.py

```diff
@@ -27,52 +19,24 @@ class EVEClient(GenericClient):
 
     >>> from sunpy.net import Fido, attrs as a
     >>> results = Fido.search(a.Time("2016/1/1", "2016/1/2"),
-    ...                       a.Instrument.eve, a.Level.zero)  #doctest: +REMOTE_DATA
+    ...                       a.Instrument.eve, a.Level("0cs"))  #doctest: +REMOTE_DATA
     >>> results  #doctest: +REMOTE_DATA
     <sunpy.net.fido_factory.UnifiedResponse object at ...>
     Results from 1 Provider:
     <BLANKLINE>
     2 Results from the EVEClient:
-         Start Time           End Time      Source Instrument Wavelength
-    ------------------- ------------------- ------ ---------- ----------
-    2016-01-01 00:00:00 2016-01-02 00:00:00    SDO        eve        nan
-    2016-01-02 00:00:00 2016-01-03 00:00:00    SDO        eve        nan
+         Start Time     Source Provider  Physobs   Instrument Level
+    ------------------- ------ -------- ---------- ---------- -----
+    2016-01-01 00:00:00    SDO     LASP irradiance        eve   0CS
+    2016-01-02 00:00:00    SDO     LASP irradiance        eve   0CS
     <BLANKLINE>
     <BLANKLINE>
 
     """
-
-    def _get_url_for_timerange(self, timerange, **kwargs):
-        """
-        Return list of URLS corresponding to value of input timerange.
-
-        Parameters
-        ----------
-        timerange: `sunpy.time.TimeRange`
-            time range for which data is to be downloaded.
-
-        Returns
-        -------
-        urls : list
-            list of URLs corresponding to the requested time range
-
-        """
-        # If start of time range is before 00:00, converted to such, so
-        # files of the requested time ranger are included.
-        # This is done because the archive contains daily files.
-        if timerange.start.strftime('%M-%S') != '00-00':
-            timerange = TimeRange(timerange.start.strftime('%Y-%m-%d'), timerange.end)
-        eve = Scraper(BASEURL)
-        return eve.filelist(timerange)
-
-    def _get_time_for_url(self, urls):
-        eve = Scraper(BASEURL)
-        times = list()
-        for url in urls:
-            t0 = eve._extractDateURL(url)
-            # hard coded full day as that's the normal.
-            times.append(TimeRange(t0, t0 + TimeDelta(1*u.day)))
-        return times
+    baseurl = (r'http://lasp.colorado.edu/eve/data_access/evewebdata/quicklook/'
```

Comment:
These are now class attributes?

Reply:
yes, only a baseurl and an extractor are needed to define most of the clients.